pyccl/concentration.py

```diff
@@ -0,0 +1,317 @@
+from . import ccllib as lib
+from .background import growth_factor
+from .hmfunc import sigmaM
+from .massdef import MassDef, mass2radius_lagrangian
+import numpy as np
+from .power import linear_matter_power
+from scipy.interpolate import InterpolatedUnivariateSpline
+
+
+class Concentration(object):
+    """ This class enables the calculation of halo concentrations.
+
+    Args:
+        name (str): a name for this c(M) object.
+        mass_def (:obj:`MassDef`): a mass definition object that fixes
+            the mass definition used by this c(M)
+            parametrization.
+    """
+    name = 'default'
+
+    def __init__(self, name, mass_def=None):
+        self.name = name
```

Comment:
Huh? Doesn't this override the class-level attribute?

Reply:
Ah damn. Yes, you're right. I've actually removed the object-level name, since I'm not using that one for anything